### ISSUE TITLE
Make `fetch_tbl` and `fetch_df` compatible with `vroom` upgrade

### ISSUE THREAD

**Issue by brookslogan** [cmu-delphi/epidatr#60]
Using either of these generates:
```
Error: The `file` argument of `vroom()` must use `I()` for literal data as of vroom 1.5.0.
```

#37 looks like it fixes this for `fetch_tbl` ~but also needs applied to `fetch_df`~ [`fetch_df` dispatches to `fetch_tbl`].

**Comment by brookslogan** [comment]
Degrading priority.  This only appears to be an error or even a warning for users when using nondefault `options(lifecycle_verbosity=.....)` settings, as it's based on `lifecycle::deprecate_soft`.  We should still address the deprecation, but it's not as urgent.

**Comment by dshemetov** [comment]
Fixed by #99
